CLogger/include/CLogMacro/CLogMacro.h

```diff
@@ -37,17 +38,20 @@
 #define F_15(F, a, b, c, d, e, f, g, h, i, j, k, l, m, n, o) F(a), F(b), F(c), F(d), F(e), F(f), F(g), F(h), F(i), F(j), F(k), F(l), F(m), F(n), F(o)
 
 // Returns var name
-#define GET_NAME(x) #x
+#define GET_NAME(x) #x \
```

Comment:
There are a lot of macros, it will be good to reduce them, also to make public only a few of them.
For example public macros: FUNCTION_START(), FUNCTION_END(), LOG_PROD(), LOG_DEBUG(), LOG_ERROR(), LOG_WARNING()
FUNCTION_START() could use LOG_DEBUG() or LOG_TRACE() depends on log level.
We can hide a try-catch block under FUNCTION_START() and FUNCTION_END() to catch exceptions.

Reply:
Interesting thought, I'll think about what can be done in the next update.

But how can I hide macros to make them "private"?

Reply:
Can I hide macros via `#undef` or I must find another way?

Reply:
#undef should help

Reply:
That's great. It'll be done.

Reply:
Not great. `#undef` annihilates almost all macros `cause they try to use "undefined" macros.

Another way is to implement helper-macros in source file, but they have to be accessible for my useful header-macros at the same time.

Reply:
Did you try to put #undef in the end of the file with macros?

Reply:
That's what I tried to do.